.github/workflows/nodejs.yml

```diff
@@ -23,38 +27,44 @@ jobs:
       OPEN_QUAD_WORD: fake-quad-word
 
     steps:
-      - name: Checkout
+      - name: ⬇️ Checkout code
         uses: actions/checkout@v3
 
-      - name: Setup Node.js
+      - name: ⎔ Setup Node.js
         uses: actions/setup-node@v3
         with:
           node-version-file: .node-version
           cache: npm
 
-      - name: Install Dependencies
+      - name: 📥 Download dependencies
         run: npm ci
 
-      - name: Test
-        run: npm run test-ci
+      - name: 🧵 Lint
+        run: npm run lint
 
-  deploy-staging:
-    name: Build and Deploy to Staging
+      - name: 🧪 Run tests
+        run: npm run test:ci
+
+  deploy-dev:
+    name: Build and Deploy to dev
     runs-on: ubuntu-latest
     needs: test
     environment:
-      name: Staging
-    if: github.ref == 'refs/heads/staging'
+      name: dev
+    if: github.ref == 'refs/heads/dev'
```

Comment:
**thought**: this will require terraform updates to the identity federation. Is that on your mind after this merges?

Reply:
I'm not sure that I thought about this but it appears that [the terraform code](https://github.com/agrc/gcp-terraform/blob/c4668441ae690f18c921d9306b4711a6366d345c/print-proxy/main.tf#L60) already configured for `dev`.